src/lib.rs

```diff
@@ -96,6 +100,72 @@ pub struct FangError {
     pub description: String,
 }
 
+/// Possible states of the task
+#[derive(Debug, Eq, PartialEq, Clone)]
+#[cfg(feature = "asynk")]
+#[derive(ToSql, FromSql, Default)]
+#[cfg(feature = "asynk")]
+#[postgres(name = "fang_task_state")]
+#[cfg(feature = "blocking")]
+#[derive(diesel_derive_enum::DbEnum)]
+#[cfg(feature = "blocking")]
+#[ExistingTypePath = "crate::schema::sql_types::FangTaskState"]
+pub enum FangTaskState {
+    /// The task is ready to be executed
+    #[cfg(feature = "asynk")]
+    #[postgres(name = "new")]
+    #[cfg(feature = "asynk")]
+    #[default]
+    New,
+    /// The task is being executing.
```

Comment:
The task is being executed

Reply:
[fix spelling and move use Uuid to top](https://github.com/ayrat555/fang/pull/126/commits/5ef6ce7fc11addd07f404fc566e564f02532c1a3)